src/lib/drivers/TSYS01.cpp

```diff
@@ -0,0 +1,177 @@
+//
+// Created by Genton Mo on 2019-07-19.
+// Ported by Alvaro
+//
+
+#include "FreeRTOS.h"
+#include "task.h"
+#include "TSYS01.h"
+#include "debug.h"
+
+TSYS01::TSYS01(SPIInterface_t* spiInterface, IOPinHandle_t *cspin):
+_SPIInterface(spiInterface),_CSPin(cspin)
+{
+  IOWrite(_CSPin, 1);
+}
+
```

Comment:
Copied over from spotter repo, all add errors to queues were replaced with printfs. Nothing else was changed